### DISCUSSION TITLE
Meeting Agenda 2/22

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#300] [Meeting Agendas/Minutes]
1. get weekly report turned in
2. discuss renaming vis packages
    - `ott-vis` -> `ott-vis-panel`
    - `ott-vis-common` -> `ott-vis`
3. discuss tasks/troubleshoot problems/unblock people
4. meet with darian

**Comment by moreno-michael** [comment]
Meeting minutes 2/22

1. Weekly report and lean canvas submitted
2. vis packages will be renamed after MVP of visualization is complete to avoid merge conflicts
3. Task update
- Issue #1295 delayed until next sprint, #1360 in progress
- Issue #1339 in progress
- Issue #1358 in progress
4. Met with Darian
- Issue #276 in progress
- Issue #301 assigned
- Issue #302 assigned
